### PR TITLE
WAC/Azure Extended Network: replace recursive URL

### LINKED ISSUE #4749: Wrong Link?
[Enter feedback here]

Azure Extended Network Link shows the same site again.
Should it follow to this site?: https://docs.microsoft.com/de-de/azure/virtual-network/subnet-extension

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7c8fa86e-65b1-d8d1-c1e6-d7ba721d7f75
* Version Independent ID: 714b296f-07d4-3e09-3928-483e9069d180
* Content: [Connecting Windows Server to Azure hybrid services](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/azure/)
* Content Source: [WindowsServerDocs/manage/windows-admin-center/azure/index.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/manage/windows-admin-center/azure/index.md)
* Product: **windows-server**
* Technology: **windows-admin-center**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As suggested in issue ticket #4749 (Wrong Link?), this is a proposal to use a link URL that actually goes somewhere, instead of reloading this page.

Also note: the recursive link is used 3 times on this very page. The URL https://go.microsoft.com/fwlink/?linkid=2109517&clcid=0x409 points to https://docs.microsoft.com/windows-server/manage/windows-admin-center/azure/ instead of the indicated page [Azure Extended Network].

Thanks to @Theragus for pointing out the issue and posting a good replacement suggestion.

**Ticket closure or reference:**

Closes #4749

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
I have limited my changes to only correcting the URL in question, so the change(s) should be fairly obvious. Feel free to use the other views if you want (even though not needed, strictly speaking).

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@illfated: I'm sorry - only the author of this article, @jasongerend, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
(also ref. #3958 )

**Comment by illfated** [comment]
Conclusion: This PR aims to correct 3 identical URLs to use the same intended target page, instead of the recursive existing URL. 
No further edits have been made in this PR so far.

**Comment by PRMerger12** [comment]
@illfated : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@illfated: I'm sorry - only the author of this article, @jasongerend, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by davannaw-msft — APPROVED** [review]

**Comment by PRMerger13** [comment]
@illfated : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by illfated** [comment]
I overlooked localization (/en-gb) in 2 of the URLs I provided in my initial PR commit. These are now corrected.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger14** [comment]
@illfated: I'm sorry - only the author of this article, @jasongerend, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: WindowsServerDocs/manage/windows-admin-center/azure/index.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ For more info, see [Deploy a Cloud Witness for a Failover Cluster](../../../fail
 - **Connect your on-premises servers to an Azure Virtual Network with [Azure Network Adapter](https://aka.ms/WACNetworkAdapter)**
 Let Windows Admin Center simplify setting up a point-to-site VPN from an on-premises server into an Azure virtual network.
 
-- **Make Azure VMs look like your on-premises network with [Azure Extended Network](https://go.microsoft.com/fwlink/?linkid=2109517&clcid=0x409)**
```
Comment (sr0784359) [inline WindowsServerDocs/manage/windows-admin-center/azure/index.md:86]: - [ ] 

- [ ]

Comment (illfated) [reply WindowsServerDocs/manage/windows-admin-center/azure/index.md:86]: If you have got anything relevant to contribute, please be more clear in your feedback.

**Pull request merged**